src/org/ods/orchestration/util/Project.groovy

```diff
@@ -362,20 +363,10 @@ class Project {
         this.data.jira.undoneDocChapters = this.computeWipDocChapterPerDocument(this.data.jira)
 
         if (this.hasWipJiraIssues()) {
-            def message = this.isWorkInProgress ?'Pipeline-generated documents are watermarked ' +
-                "'${LeVADocumentUseCase.WORK_IN_PROGRESS_WATERMARK}' " +
-                'since the following issues are work in progress: ':
-                "The pipeline failed since the following issues are work in progress (no documents were generated): "
-
-            this.getWipJiraIssues().each { type, keys ->
-                def values = keys instanceof Map ? keys.values().flatten() : keys
-                if (!values.isEmpty()) {
-                    message += '\n\n' + type.capitalize() + ': ' + values.join(', ')
-                }
-            }
+            String message = generateWIPIssuesMessage()
```

Comment:
If you use only the message inside the if, create it inside

Reply:
If you check the whole code as in the diff looks different it is inside the if block :)